### DISCUSSION TITLE
¡AYUDA! 😫 CHAMILO NO ME PERMITE CREAR CATEGORIAS DE CURSOS

### DISCUSSION THREAD

**Discussion by SoyLusho28** [chamilo/chamilo-lms#5686] [General] [answered]
Hola buen dia

Instalé totalmente desde cero mi aula de chamilo, pero al intentar crear categorias de cursos, la pagina crashea y no me permite hacer nada, ¿alguien podria ayudarme a buscar una posible solucion?

![image](https://github.com/user-attachments/assets/1bfd562e-4f64-46b3-b4cc-6768cf308392)

![image](https://github.com/user-attachments/assets/182394e3-0533-4fe6-b8eb-7ebafb6e8d70)

**Comment by Kaneda-1** [comment, marked as answer]
Hola @SoyLusho28 
revisa esta tarea https://github.com/chamilo/chamilo-lms/issues/4916
Está corregido aquí https://github.com/christianbeeznest/chamilo-lms/commit/48f192df355262034ef186870b1d53a47e5dfdc5
Saludos.

**Reply by SoyLusho28** [reply]
Hola @Kaneda-1 gracias por responder, al parecer solo era añadir esa linea, muchas gracias.!
